src/eval.c

```diff
@@ -728,6 +731,25 @@ NULL
             addReplyError(c, "Use SCRIPT DEBUG YES/SYNC/NO");
             return;
         }
+    } else if (c->argc > 2 && !strcasecmp(c->argv[1]->ptr, "dump")) {
+        int j;
+        dictEntry *de;
+        luaScript *ls;
+
+        addReplyArrayLen(c, c->argc - 2);
+        for (j = 2; j < c->argc; j++) {
+            if (sdslen(c->argv[j]->ptr) != 40) {
+                addReply(c, shared.null[c->resp]);
+                continue;
```

Comment:
This would ideally be an error instead of just silently continuing, to indicate to the user that they did something wrong. Right now they'll just the same error as if it wasn't present.